homeassistant/components/tibber/sensor.py

```diff
@@ -671,6 +671,20 @@ async def _async_update_data(self) -> None:
                 self.hass.config_entries.async_reload(self.config_entry.entry_id)
             )
 
+    async def get_prices(self, tibber_home: tibber.TibberHome) -> dict:
+        """Get price forecasts."""
+        data = tibber_home.info["viewer"]["home"]["currentSubscription"]["priceInfo"]
```

Comment:
"data" isn't a very descriptive name. How about "prices"?
```suggestion
        prices = tibber_home.info["viewer"]["home"]["currentSubscription"]["priceInfo"]
```